dex/src/main/scala/com/wavesplatform/dex/api/websockets/actors/WsHandlerActor.scala

```diff
@@ -104,7 +109,7 @@ object WsHandlerActor {
                 case subscribe: WsAddressSubscribe =>
                   subscribe.validate(settings.jwtPublicKey, AddressScheme.current.chainId) match {
                     case Left(e) =>
-                      context.log.debug(s"WsAddressSubscribe(k=${subscribe.key}, t=${subscribe.authType}) failed with ${e.code}")
+                      context.log.debug(s"WsAddressSubscribe(k=${subscribe.key}, t=${subscribe.authType}) failed with ${e.message}")
```

Comment:
I think its better to use `e.message.text`

Reply:
Fixed :)